dds/DCPS/XTypes/TypeLookupService.h

```diff
@@ -56,8 +56,8 @@ class OpenDDS_Dcps_Export TypeLookupService : public virtual DCPS::RcObject {
   void get_type_dependencies_i(const TypeIdentifierSeq& type_ids,
     TypeIdentifierWithSizeSeq& dependencies) const;
 
-  // Only minimal Type Objects for now
-  TypeMap minimal_type_map_;
+  // Contains both minimal and complete type mapping.
```

Comment:
Confirm that the type identifiers for minimal and complete will not clash.

Reply:
The TypeIdentifier kinds are considered in `operator<` of TypeIdentifier. Minimal and complete type identifiers have different kinds (EK_MINIMAL vs. EK_COMPLETE) so they won't clash. In case both minimal and complete types have kind TI_STRONGLY_CONNECTED_COMPONENT, their TypeObjectHashId's kinds will be different (EK_MINIMAL vs. EK_COMPLETE). So I think it is safe to use a single map for both minimal and complete types.